templates/layouts/bootstrap.html.ep

```diff
@@ -40,6 +40,7 @@
 
   </head>
   <body>
+    <div id="nav-main">
```

Comment:
why do you need a div around the nav ? and why do you need a id anyway? There aren't that many 'nav' elements around